astropy/utils/introspection.py

```diff
@@ -111,6 +121,19 @@ def minversion(module, version, inclusive=True, version_path='__version__'):
     True
     """
 
+    if isinstance(module, types.ModuleType):
+        module_name = module.__name__
+    elif isinstance(module, six.string_types):
+        module_name = module
+        try:
+            module = resolve_name(module_name)
+        except ImportError:
+            return False
```

Comment:
See my comment in #3478 - I do think you should let the import error be raised. If some package needs it in their setup, then the import error should actually happen.  Note that this should never be a problem: before we just imported numpy during setup as well to construct the various `NUMPY_LT_...` strings.